src/main/kotlin/com/hedvig/rapio/qa/dto/UnsignMemberResponse.kt

```diff
@@ -0,0 +1,5 @@
+package com.hedvig.rapio.qa.dto
+
+data class UnsignMemberResponse(
+    val isUnsigned:Boolean
```

Comment:
Format on save, or Option-Cmd-L
```suggestion
    val isUnsigned: Boolean
```